www/src/pages/foundations/spacing.tsx

```diff
@@ -54,22 +58,24 @@ export type SpaceBlockTypes = {
   utilityClass: string,
 };
 
-const SpaceBlock = ({ utilityClass }: SpaceBlockTypes) => (
-  <code
-    className={classNames(utilityClass)}
-    style={{
-      display: 'flex',
-      alignItems: 'center',
-      justifyContent: 'center',
-      height: '5rem',
-      textAlign: 'center',
-      width: '10rem',
-      background: 'rgba(0,0,0,.1)',
-    }}
-  >
-    {utilityClass ? `.${utilityClass}` : null}
-  </code>
-);
+function SpaceBlock({ utilityClass }: SpaceBlockTypes) {
```

Comment:
This refactoring is related to: ESLint: Function component is not a function declaration(react/function-component-definition)